run_aws_linker.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
Can we move this file into the `splink3_demos` branch of the `splink_demos` repo before we merge

Reply:
Yeah, I think that makes sense. This `.ipynb` script was never meant to stick around once we merged. It was simply to allow for wider testing.